File: Exemplos/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Java-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/TERMICA/ImprimeImagemMemoria.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA

import com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.TermicaCommand

class ImprimeImagemMemoria(private val key: String, private val scala: Int) :
    TermicaCommand("ImprimeImagemMemoria") {
    override fun functionParameters(): String {
        return "\"key\"" + ":" + "\"" + key + "\"" + "," +
                "\"scala\"" + ":" + scala
    }
}

File: Exemplos/App_IntentDigitalHUB_ReactNative/Elgin_IntentDigitalHUB_ReactNative-font/src/intentDigitalHubService/TERMICA/ImprimeImagemMemoria.ts
import {TermicaCommand} from './TermicaCommand';

export class ImprimeImagemMemoria extends TermicaCommand {
  readonly key: string;
  readonly scala: number;

  constructor(key: string, scala: number) {
    super('ImprimeImagemMemoria');
    this.key = key;
    this.scala = scala;
  }

  functionParameters(): object {
    return {key: this.key, scala: this.scala};
  }
}
